docs/5.hooks.md

```diff
@@ -0,0 +1,75 @@
+# Using Action and Filter Hooks
+
+The Abilities API provides action and filter hooks that allow developers to customize and extend its functionality. Below are the available hooks:
+
+## Quick Links
+
+- [Actions](#actions)
+- [Filters](#filters)
+  - [`wp_ability_args`](#wp_ability_args)
+
+## Actions
+
+There are currently **no Action hooks** provided by this version of the Abilities API.
+
+## Filters
+
+### `wp_ability_args`
+
+> [!IMPORTANT]
+> This filter is prefixed with `wp_` to avoid potential naming collisions.
+> Once merged into WordPress core, the prefix will likely be removed to preserve backward-compatibility.
```

Comment:
I would go with the final name from the start to avoid the hassle. `register_ability_args` should be distinct enough.